src/include/cart/api.h

```diff
@@ -2375,6 +2376,15 @@ int crt_context_quota_limit_get(crt_context_t crt_ctx, crt_quota_type_t quota, i
 int
 crt_req_get_proto_ver(crt_rpc_t *req);
 
+/**
+ * Get the rpc origin address.
+ *
+ * \param[in] rpc		pointer to RPC request
+ * \return			the origin address of the RPC
+ */
+char *
+crt_rpc_get_origin_addr(crt_rpc_t *rpc);
```

Comment:
The api functions follow format of crt_[module]_[verb]  (see crt_context_quota_limit_get, crt_context_create and others)
Also in public APIs we refer to 'rpc' as 'req'  (e.g. crt_req_create(), crt_req_get_proto_ver() etc) in order to differentiate with crt_rpc_ functions that are internal.
So based on those 2 things, this should be renamed to something like crt_req_origin_addr_get()

Reply:
ah, ok